iree/turbine/kernel/wave/scheduling/loop_reconstruction_utils.py

```diff
@@ -227,11 +249,11 @@ def liveness_analysis(
             logger.debug(
                 f"Node: {node}, User: {user.fx_node}, lifetime: {user.scheduling_parameters['stage'] - custom.scheduling_parameters['stage']}"
             )
-            lifetime[node] = max(
+            user_lifetime = (
```

Comment:
is there a purpose to this refactoring aside from styling?

Reply:
I was doing something with it, but dropped and decided to keep it like this for styling.